robotpy_installer/cli_init.py

```diff
@@ -36,3 +36,10 @@ def run(self, main_file: pathlib.Path, project_path: pathlib.Path):
             pyproject.write_default_pyproject(project_path)
 
             logger.info("Created %s", pyproject_path)
+
+        # Create .gitignore if it doesn't already exist
+        pyproject_path = pyproject.gitignore_path(project_path)
```

Comment:
This should be `gitignore_path`, but everything else seems fine and can be merged after that.